lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,34 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $characterMask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $characterMask = null, $mode = 'both')
+{
+    if (empty($characterMask)) {
+        $characterMask = " \t\n\r\0\x0B";
```

Comment:
If someone passes "empty string" instead of null as the first parameter, I believe it will override the default mask. So this catches all of those possibilities.